src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Web/Pages/_Layout.cshtml

```diff
@@ -47,6 +48,17 @@
     {
         <link rel="manifest" href="manifest.json">
     }
+    @if (isElectron)
+    {
+        <style>
+            * {
```

Comment:
plz move these lines to the below style tags.

Reply:
It's a VS/Framework limitation